### ISSUE TITLE
Deprecate Windows 8 tile icons

### ISSUE THREAD

**Issue by Rotzbua** [itgalaxy/favicons#414]
## Problem

The title are only used on Windows 8 and Internet Explorer. Both are end of life.
* https://learn.microsoft.com/en-us/lifecycle/products/windows-81
* https://learn.microsoft.com/en-us/lifecycle/products/internet-explorer-11

## Solution

* Deprecate the option

**Comment by andy128k** [comment]
These icons and manifests are used by modern versions of Windows too. E.g. [Pin to Start](https://www.winhelponline.com/blog/pin-website-shortcut-start-screen-windows-10/).

**Comment by Rotzbua** [comment]
Yes it is still possible to pin to start with windows. But the underling technique are changed.
Microsoft is now are more committed to standards and PWAs.
They launched the test and support page https://www.pwabuilder.com/ .

If you dig into the documentation, you will finally find the following document with the corresponding resolutions:
https://learn.microsoft.com/en-us/microsoft-edge/progressive-web-apps-chromium/how-to/icon-theme-color
